pkg/lint/rules/values_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rules
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+const (
+	badValuesDir = "testdata/badvaluesfile"
+)
+
+var (
+	badValuesFilePath         = filepath.Join(badValuesDir, "values.yaml")
+	nonExistingValuesFilePath = filepath.Join("/fake/dir", "values.yaml")
+)
+
+func TestValidateValuesYamlNotDirectory(t *testing.T) {
+	_ = os.Mkdir(nonExistingValuesFilePath, os.ModePerm)
+	defer os.Remove(nonExistingValuesFilePath)
+
+	err := validateValuesFileExistence(nonExistingValuesFilePath)
+	if err == nil {
+		t.Errorf("validateValuesFileExistence to return a linter error, got no error")
+	}
+}
+
+func TestValidateValuesName(t *testing.T) {
+	err := validateValuesFile(badValuesFilePath)
+	if err == nil {
+		t.Errorf("validateValuesFile to return a linter error, got no error")
+	}
+}
```

Comment:
This is tested already here: https://github.com/helm/helm/blob/master/pkg/lint/lint_test.go#L91

Reply:
I am sorry for interruption. I have removed this test :)